Mrs_Cake/Services/UserService.cs

```diff
@@ -29,10 +29,14 @@ public User Create(User user)
             return user;
         }
 
-        public void Update(string id, User user) =>
-
+        public void Update(string id, User user)
```

Comment:
here you'd better make if statement which checks if it is a bakery or customer to change different properties

Reply:
but we have bakery table so we can change there